### PR TITLE
Getting Started docs

### PR DESCRIPTION
**Pull request by sowmyaraj92**
Updated the screenshots and flow for the 3 pages below:
* **Introduction** - /docs/getting-started/introduction/
* **Discovering templates** - /docs/getting-started/importing-templates/
* **Settings** -/docs/getting-started/settings/

### CODE REVIEW

**Review by jerry-li-dev — CHANGES_REQUESTED** [review]
Line 20 needs to be changed to link to Publishing public API
Line 21 needs to be changed to link to "/docs/publishing-your-api/publish-public-api/"

File: src/pages/docs/getting-started/importing-templates.md [hunk 1]
```diff
@@ -25,36 +25,37 @@ warning: false
```
Comment (jerry-li-dev) [inline src/pages/docs/getting-started/importing-templates.md:25]: Creating Templates and API Network features will be removed in v8. We need to remove this page or revise to cover how to import projects from public workspaces from [https://www.postman.com/explore](https://www.postman.com/explore)

The contextual links should include a link to `/docs/publishing-your-api/publish-public-api/` and `/docs/collaborating-in-postman/using-workspaces/creating-workspaces/`

We will need to remove 
`  name: "Publishing to the API Network" `
`  url: "/docs/publishing-your-api/add-api-network/" `

**Review by jerry-li-dev — CHANGES_REQUESTED** [review]

File: src/pages/docs/getting-started/importing-templates.md (hunk 1 above)
Comment (sowmyaraj92) [reply src/pages/docs/getting-started/importing-templates.md:25]: I have removed the `Publishing to API network` link on the right side. We can revise this page and add how to import from public workspaces.

**Comment by claireriley** [comment]
@sowmyaraj92 @jerry-li-dev let's get this merged asap as it include the opening page of the docs

**Review by jerry-li-dev — APPROVED** [review]

**Comment by jerry-li-dev** [comment]
Lgmt

**Pull request merged**